### PR TITLE
Fix typos in documentation [12519]

### PR DESCRIPTION
**Pull request by juanlofer-eprosima**
Fix typos and add details for installation from sources in Linux.

### CODE REVIEW

File: docs/rst/getting_started/entities.rst [hunk 1]
```diff
@@ -103,14 +103,14 @@ Logical Entities
...
+*Domain* represents a logical abstraction in DDS protocol that divides the DDS network into partitions,
 making each *Domain* completely independent and unaware of others.
 This logical partition depends on the chosen discovery protocol.
 
-In case of using the *Simple Discovery Protocol* (discovery protocol by default in Fast DDS) as the default discovery
-mechanism, the *Domain* will be represented by a number, and every DDS entity in that *Domain* is going to discover
-the rest entities deployed on the same *Domain*.
-In case of using *Discovery Server* as discovery protocol the partition will be made by the *Discovery Server*
+In case of using *Simple Discovery Protocol* (discovery protocol by default in Fast DDS) as the default discovery
+mechanism, the *Domain* will be represented by a number, and every DDS entity in that *Domain* will discover
```
Comment (jparisu) [inline docs/rst/getting_started/entities.rst:111]:
```suggestion
mechanism, the *Domain* will be represented by a number (domain ID), and every DDS entity in that *Domain* will discover
```

File: docs/rst/getting_started/entities.rst [hunk 2]
```diff
@@ -140,25 +140,25 @@ Physical Entities
 
 Host
 ----
-*Host* make reference to the physical (or logical i.e. Docker) machine where one or some DDS *Processes*
+A *Host* makes reference to the physical (or logical i.e. Docker) machine where one or more DDS *Processes*
```
Comment (jparisu) [inline docs/rst/getting_started/entities.rst:143]: Add this `A` to every definition or do not add it here.

File: docs/rst/getting_started/entities.rst [hunk 3]
```diff
@@ -140,25 +140,25 @@ Physical Entities
...
 are running.
 This entity is connected directly with the *Users* running in this *Host*.
 
 .. _user_entity:
 
 User
 ----
-*User* make reference to the different users that could run in a *Host*.
+A *User* makes reference to the different users that could run in a *Host*.
```
Comment (jparisu) [inline docs/rst/getting_started/entities.rst:151]: Add this `A` to every definition or do not add it here.

File: docs/rst/getting_started/entities.rst [hunk 4]
```diff
@@ -140,25 +140,25 @@ Physical Entities
...
-This entity is connected directly with the *Host* it belongs, and the running *Processes* that are being executed
+This entity is connected directly with the *Host* it belongs to, and the running *Processes* that are being executed
 within this *User*.
 
 .. _process_entity:
 
 Process
 -------
-*Process* represents each process running an application using *Fast DDS*. Be aware that it is possible to
+A *Process* represents each process running an application using *Fast DDS*. Be aware that it is possible to
```
Comment (jparisu) [inline docs/rst/getting_started/entities.rst:160]: Add this `A` to every definition or do not add it here.

**Review by jparisu — CHANGES_REQUESTED** [review]

File: docs/rst/getting_started/entities.rst (hunk 4 above)
Comment (juanlofer-eprosima) [reply docs/rst/getting_started/entities.rst:160]: :+1:

File: docs/rst/getting_started/entities.rst (hunk 3 above)
Comment (juanlofer-eprosima) [reply docs/rst/getting_started/entities.rst:151]: :+1:

File: docs/rst/getting_started/entities.rst (hunk 2 above)
Comment (juanlofer-eprosima) [reply docs/rst/getting_started/entities.rst:143]: :+1:

File: docs/rst/getting_started/entities.rst (hunk 1 above)
Comment (juanlofer-eprosima) [reply docs/rst/getting_started/entities.rst:111]: :+1:

File: docs/rst/getting_started/entities.rst [hunk 5]
```diff
@@ -26,10 +26,10 @@ i.e. one *DomainParticipant* is related with its *User* by the relation with its
 DDS Entities
 ============
 
-This entities represent the DDS entities that manage the communication. That is, the *DomainParticipants*
-and the DataWriters and DataReaders it manages.
-Each DataReader/DataWriter has one or more associated Locator entities.
-Locators are the network addresses through which DataReaders/DataWriters communicate in a DDS network.
+These entities represent the DDS entities that manage the communication. That is, the *DomainParticipants*
+and the *DataWriters* and *DataReaders* it manages.
```
[the remark this answers is not in the document]
Comment (juanlofer-eprosima) [reply docs/rst/getting_started/entities.rst:30]: Not sure about this one. I would leave the "it manages" or delete an "and", i.e. "That is, the DomainParticipants, DataWriters and DataReaders".

File: docs/rst/developer_manual/installation/sources/linux.rst [hunk 6]
```diff
@@ -191,6 +195,11 @@ Colcon installation
 
         colcon build --cmake-args -DQT_PATH=<qt-installation-path>
 
+    .. note::
+
+        If compiling *Fast DDS* with colcon, the statistics module must be activated by adding the flag
+        :code:`-DFASTDDS_STATISTICS=ON`
```
[the remark this answers is not in the document]
Comment (juanlofer-eprosima) [reply docs/rst/developer_manual/installation/sources/linux.rst:201]: I would leave this part colcon specific, and add the statistics flag to the CMake installation section.
Good idea to add the reference link.

File: docs/rst/getting_started/entities.rst (hunk 5 above)
[the remark this answers is not in the document]
Comment (jparisu) [reply docs/rst/getting_started/entities.rst:30]: True. I prefer "That is, the DomainParticipants, DataWriters and DataReaders", but it is okey like that.

[the remark this answers is not in the document]
Comment (juanlofer-eprosima) [reply docs/rst/getting_started/entities.rst:30]: :+1:

[the remark this answers is not in the document]
Comment (jparisu) [reply docs/rst/getting_started/entities.rst]: DomainParticipant is the subject of "it manages", so use the old version   "That is, the *DomainParticipants*
and the *DataWriters* and *DataReaders* it manages" or the one without manages "That is, the *DomainParticipants*,
*DataWriters* and *DataReaders*" (I prefer this last one)

File: docs/rst/getting_started/entities.rst (hunk 2 above)
Comment (jparisu) [reply docs/rst/getting_started/entities.rst]: NIT: Could you change Locator and DomainParticipant definitions so they start with *<entity>* as the others?

**Review by jparisu — CHANGES_REQUESTED** [review]
Small details of old suggestions

File: docs/rst/getting_started/entities.rst (hunk 2 above)
Comment (juanlofer-eprosima) [reply docs/rst/getting_started/entities.rst]: :+1:

File: docs/rst/getting_started/entities.rst (hunk 5 above)
[the remark this answers is not in the document]
Comment (juanlofer-eprosima) [reply docs/rst/getting_started/entities.rst]: :+1:

**Review by jparisu — APPROVED** [review]
LGTM

**Pull request merged**